### DISCUSSION TITLE
该以一种怎么样的心态来对待开源 - TEST

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#157] [General]
# 该以一种怎么样的心态来对待开源 - TEST

Hello World!Share最近好几位我喜欢的开发者被开源工作影响到了心情，加上国内的开源环境氛围不是很好，想碎碎念来聊一下，对于开发者和使用者，我们应该以一种怎么样的心态来对待开源，让自己可以持续的快乐写自己喜欢的代码。可能有些我理解偏差地方，很欢迎大伙指出。

http://localhost:4000/2023-09-18/open.html

**Comment by heroes007** [comment]
123
